tests/istio.mk

```diff
@@ -48,13 +48,13 @@ e2e_bookinfo_trustdomain: build test/local/auth/e2e_bookinfo_trustdomain
 e2e_simple_run:
 	go test -v -timeout ${E2E_TIMEOUT} ./tests/e2e/tests/simple -args --auth_enable=true \
 	--egress=false --ingress=false \
-	--valueFile test-values/values-e2e.yaml \
+	--valueFile tests/e2e/values-e2e.yaml \
 	--rbac_enable=false --cluster_wide ${E2E_ARGS} ${T} ${EXTRA_E2E_ARGS}
 
 e2e_simple_noauth_run:
 	go test -v -timeout ${E2E_TIMEOUT} ./tests/e2e/tests/simple -args --auth_enable=false \
 	--egress=false --ingress=false \
-	--valueFile test-values/values-e2e.yaml \
+	--valueFile tests/e2e/values-e2e.yaml \
```

Comment:
Does this need the `test-values` directory added? Same on line 51.

Reply:
You're right. In a previous iteration I only moved them into `tests/e2e`

Reply:
Done